apps/web-tools/trustchain/components/AppAccountsSync.tsx

```diff
@@ -226,13 +236,29 @@ export default function AppAccountsSync({
 
   return (
     <div>
+      {error ? (
+        <div style={{ padding: 10, color: "red" }}>{error.message}</div>
+      ) : timestamp ? (
+        <div
+          style={{
+            textAlign: "center",
+          }}
+        >
+          Synced <Tick timestamp={timestamp} />.
```

Comment:
Could you provide translations, please?

Reply:
we don't have translations for web-tools.

Reply:
Sorry, I thought we were on the app